### PR TITLE
Nuclear Operatives now spawn with an EMP proof explosive implant

### LINKED ISSUE #10899: Ions have a chance to explode E implants.
(WEB REPORT BY: yred REMOTE: 172.93.109.202:7777)

> Revision 83bbaa3734688ba073d8098a0ba5ae2c4e64339a
> 
> General description of the issue
> Title.
> 
> What you expected to happen
> Not to be fuckered over by such a silly thing as a nuclear operative.
> 
> What actually happened
> Ops keep fucking blowing themselves up with their ion rifles, maybe moreso than the bloody RPG.
> 
> Steps to reproduce if possible
> Ion someone with an E implant.

### PR DESCRIPTION
**Pull request by JustSumBody**
closes #10899 
I think it's silly just because there's no indication the implant would even go off from an EMP, but feel free to debate it.

### CODE REVIEW

File: code/game/objects/items/weapons/implants/implant.dm [hunk 1]
```diff
@@ -198,6 +198,11 @@ Implant Specifics:<BR>"}
 			explosion(get_turf(imp_in), -1, -1, 2, 3, 3)
 			qdel(src)
 
+/obj/item/weapon/implant/explosive/nuclear //emp proof implant for nuclear operatives
```
Comment (PJB3005) [inline code/game/objects/items/weapons/implants/implant.dm]: This line isn't needed, the `emp_act` override is enough to define the type.

**Comment by Kurfursten** [comment]
You're really coming into your own with coding, keep up the good work

**Comment by ScrambledSteak** [comment]
I will miss the days of dumbass nuke ops ion blasting one of their teammates and killing everybody before they could even get to the captain. That shit was the bomb.

**Comment by Kammerjunk** [comment]
Fewer adminhelps of "This guy just shot me with an ion rifle and I exploded??" at 12:06 and not quite the same potential of instant op removal (as opposed to stun first, then remove op - don't ions also have a 3x3 effect when hitting someone?). I approve.

@Aurantiaco I'm afraid this isn't way back when. Some guy did it like yesterday or the day before.

**Comment by ghost** [comment]
Thanks for doing this. I feel sorry for the poor infiltrator op that uses EMP's in maint killholes and blow themselves up doing it.

**Comment by Kurfursten** [comment]
@Kammerjunk They do, I have seen an entire nuke op team wiped out a few months back on Deff when one shot a borg at melee range with an ion rifle. The AOE blew up 2 ops, the explosion radius caused the remaining two to fall over and start bleeding.

To add insult to injury, it didn't even stop the borg because ion rifles do joke-tier stun duration and energy drain. They all got cuffed and the round was over.

**Comment by icantthinkofanameritenow** [comment]
And I remember more than one occasion when I used an EMP grenade to try and disable a borg, only to get robusted and wonder why the *deathgasp wasn't doing anything. I support this.

**Comment by ghost** [comment]
Ion's are what, 1 hvy 2 light? EMP has a 50% chance to be a light effect when it's on the border range so maybe extended the range would help so you get that heavier stun.

**Comment by icantthinkofanameritenow** [comment]
@SarahJohnson489

Are you still talking about the implants or something else?

**Comment by Kurfursten** [comment]
Nah, Sarah is referring to the ion rifle area of effect with regards to the cyborg stun effect

It's pretty unnoticeable right now but that's for a different balance PR

**Pull request merged**